client/map/storeLocator.html

```diff
@@ -1,102 +1,102 @@
+                        <option>5 KM</option>
+                        <option>10 KM</option>
+                    </select>
+                </div>
+                <div>
+                    Medicament dans la pharmacie
+                    <input type="text" name="map-drug-gist" value="{{enteredDrugsList}}" id="drug_short_list" readonly>
+                    <img src="images/closer.png" class="clearfield cleartableau">
+                </div>
+                <div class="tableau" id="drug_long_list">
+                    <div>
+                        <span>Nom de médicaments</span>
+                        <span>Quantité</span>
+                        <img src="images/closer.png" class="closer" data-close="drug_long_list">
+                    </div>
+                    {{#each medicines}}
+                        <div>
+                            <img src="images/remove.png" class="rowremover">
+                            <span>{{name}}</span>
+                            <span>{{quantity}}</span>
+                        </div>
+                    {{/each}}
+                    <form class="search-form">
+                        {{> inputAutocomplete settings=settings id="search-box" class="form-control" placeholder="Nom de l'officine..." autocomplete="off"}}
+                        <input type="text" id="quantity" placeholder="Quantité..." autocomplete="off" />
+                        <input type="submit" value="" />
+                    </form>
+                </div>
+                <div>
+                    <p>Attention ! Le médicamant n'a pas été trouvé. Veullez verifier le nom du médicament saisit.</p>
+                </div>
+            </section>
+            <article class="map-container">
+                {{> googleMaps}}
+            </article>
+            <aside>
+                {{> storesList}}
+            </aside>
+        </div>
+    </div>
 </template>
 
 <template name="storesList">
-	<div id="search_info">
-		<img src="images/list_icon.png" >
-		{{stores.length}} Stores found for the following criteria<br><br>
-		<b>DISTANCE</b> - Any<br>
-		<b>DRUGS</b> - {{enteredDrugsList}}<br>
-	</div>
-	{{#each stores}}
-	  {{> store}}
-	{{/each}}
+    <div id="search_info">
+        <img src="images/list_icon.png" >
+        {{stores.length}} Pharmacie(s) trouvée(s) avec les critères suivant<br><br>
+        <b>DISTANCE</b> - Tous<br>
+        <b>MÉDICAMENT</b> - {{enteredDrugsList}}<br>
+    </div>
+    {{#each stores}}
+        {{> store}}
+    {{/each}}
 </template>
 
 <template name="store">
-	<div class="stores_list">
-		<p>
-			<b>{{storeName}}</b><br>
-			{{address.street}}<br>
-			{{address.postalCode}} {{address.city}}, {{address.country}}<br>
-			<a href="mailto:support@store.com">support@store.com</a><br>
-			<a href="tel:+918041717222">{{telephone}}</a><br>
-			<!-- {{stock}} -->
-		</p>
-		<button>ORDER HERE</button>
-	</div>
+    <div class="stores_list">
+        <p>
+            <b>{{name}}</b><br>
```

Comment:
Please change name to storeName. 
*Note: I think you probably worked on an earlier version of the project which was using name instead of storeName*